silhouette/test/com/mohiva/play/silhouette/impl/providers/BasicAuthProviderSpec.scala

```diff
@@ -135,15 +150,29 @@ class BasicAuthProviderSpec extends PlaySpecification with Mockito {
     lazy val fooHasher = {
       val h = mock[PasswordHasher]
       h.id returns "foo"
+      h.isSuitable(any) answers { p =>
```

Comment:
Does the mocking library make this necessary? The default implementation doesn't get used? It really doesn't know what the input type to `isSuitable` is? If you actually have to redefine it can I suggest `p.asInstanceOf[PasswordInfo].hasher == "foo"`?